releasenotes/notes/type-checking-in-operator-1a65545c4ead8b1e.yaml

```diff
@@ -0,0 +1,11 @@
+---
+upgrade:
+  - |
+    If you use the ``sum`` function to add stop strategies, you must initialize the sum with ``wait_none()``.
+fixes:
+  - |
+    Fixes #211.
```

Comment:
I agree. I think it's even better to remove the `fixes` section entirely since #211 is not really a bug. The changes are described in length in the `other` section.